pkgs/tools/networking/ookla-speedtest/default.nix

```diff
@@ -4,7 +4,7 @@ let
   pname = "ookla-speedtest";
   version = "1.1.1";
```

Comment:
I’m not sure whether to change this to 1.1.1.84. macOS is the only platform where the latest version is 1.1.1.84; all the others is 1.1.1.